hpt/__init__.py

```diff
@@ -0,0 +1,278 @@
+from typing import Dict, Optional, List, Any, Callable
+from enum import Enum, unique
+import numpy as np
+from hyperopt import fmin, tpe, hp
+
+from deephub.common.io import load_yaml, AnyPathType
+from deephub.common.utils import hash_from_dict
+from pathlib import Path
+
+
```

Comment:
Can you please explain this to me? We had discussed to use it only in case we understand it. Did you found out why we need this?